### ISSUE TITLE
Driving Semantic Parsing from the World’s Response

### ISSUE THREAD

**Issue by Tahkare** [Tahkare/PhD-archive#5]
## TL;DR
They propose a way of training a semantic parser without having access to the expected program. For each example, they generate the most likely program and ask a evaluator if the output of that program is correct. If it is, they consider that program to be correct for the given input and add it to their training set.

### Article Link
https://www.aclweb.org/anthology/W10-2903.pdf

### Author
J. Clarke, D. Goldwasser, M.W. Chang & D.Roth

## Key Takeaways
* Quite efficient way to reduce supervision
* However, you need an evaluator

## Comments/ Questions
Personal reflexion : their two-step procedure of alignment and then dependency parsing is interesting. We could try to see if we could make a slightly different alignement procedure (check 1-to-1/1-to-N/N-to-1) before computing dependencies ?
